app/frontend/components/pages/login/loginPage.tsx

```diff
@@ -155,7 +155,7 @@ const SubCardByAuthMethod = ({authMethod}: {authMethod: AuthMethodType}) => {
 
 const NuFiBannerWrapper = () => (
```

Comment:
naming nitpick - in code I think we generally refer to `NuFi` as `Nufi`
```suggestion
const NufiBannerWrapper = () => (
```